### PR TITLE
Portfolio dom

### PR DESCRIPTION
**Pull request by osbentech**
here I and my coding partner manipulated the menu bar using js

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,14 +15,22 @@
...
+             <button class="btn-menu"><img src="photo/navs/Union.png" alt=""></button> 
+            </div>
+            <div class="t-menu">
+              <button class="c-button">&times;</button>
+              <ul class="tul">
+                <li class="tul-1">Portfolio</li>
+                <li class="tul-1">About</li>
+                <li class="tul-1">Contact</li>
+              </ul>
+            </div>
```
Comment (Reem-lab) [inline index.html]: - [ ] As I can see here your menu doesn't cover all the section so kindly stick with the Figma template as much as possible so kindly cover all the section to match the Figma ✅💪🏻


- Your design
image

- Figma template
image

Comment (Reem-lab) [inline index.html]: - [ ] As I can see here your menu doesn't cover all the section so kindly stick with the Figma template as much as possible so kindly cover all the section to match the Figma ✅💪🏻


- Your design
image

- Figma template
image

File: index.html [hunk 2]
```diff
@@ -15,14 +15,22 @@
...
-            </nav>
+            <div class="menu">
+             <button class="btn-menu"><img src="photo/navs/Union.png" alt=""></button> 
+            </div>
+            <div class="t-menu">
+              <button class="c-button">&times;</button>
+              <ul class="tul">
+                <li class="tul-1">Portfolio</li>
+                <li class="tul-1">About</li>
+                <li class="tul-1">Contact</li>
```
Comment (Reem-lab) [inline index.html]: - [ ] Please I think that the portfolio link should take the user to the recent works sections, so kindly fix this. 😁 you can wrap it with `a` tag and include href to the sections with the id 💪🏻💪🏻💪🏻✅✅

Comment (Reem-lab) [inline index.html]: - [ ] Please I think that the portfolio link should take the user to the recent works sections, so kindly fix this. 😁 you can wrap it with `a` tag and include href to the sections with the id 💪🏻💪🏻💪🏻✅✅ and please it should take the user to the specific section and the menu should be closed after presssing on it ✅✅💪🏻

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi Team, 👋🏻👋🏻👋🏻

Good job so far!✅🎉🎉
There are some issues that you still need to work on to go to the next project but you are almost there!💪🏻💪🏻💪🏻


## Highlights ✨✨
Great job setting up the correct linters ✔️
Highly organized and clean code ✔️ 👨🏻‍💻
Professional documentation ✔️
Correct workflow ✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Reem-lab) [reply index.html]: [ Required ] Great job here but it still doesn't work, As I said here I think you shoud reference to each section with `id` so the first thing you should put in each section an `id` then 

[ Hint] you can put `href` to each `id` like this

image


- and for each section pass the id
image

- And please after taking to the specific section the menu should be closed so make sure to close it

File: index.html [hunk 3]
```diff
@@ -15,14 +15,22 @@
...
-            </nav>
+            <div class="menu">
+             <button class="btn-menu"><img src="photo/navs/Union.png" alt=""></button> 
+            </div>
+            <div class="t-menu">
+              <button class="c-button">&times;</button>
+              <ul class="tul">
+                <li><a href="tul-1">Portfolio</a></li>
+                <li><a href="tul-1">About</a></li>
+                <li><a href="tul-1">Contact</a></li>
```
Comment (Reem-lab) [inline index.html]: [ Required ] Great job here but it still doesn't work, As I said here I think you shoud reference to each section with `id` so the first thing you should put in each section an `id` then 

[ Hint] you can put `href` to each `id` like this

image


- and for each section pass the id
image

- And please after taking to the specific section the menu should be closed so make sure to close it

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi Team, 👋🏻👋🏻

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Great job setting up the correct linters ✔️
Highly organized and clean code ✔️ 👨🏻‍💻
Great job working on the requested changes ✅✅


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  menuList.classList.toggle('active');
+});
+
+close.addEventListener('click', () => {
+  menuList.classList.remove('active');
+});
+
+lists.addEventListener('click', () => {
+  menuList.classList.remove('active');
+});
```
Comment (muneebulrehman) [inline script.js]: The mobile nav menu should close automatically when I click a link for the menu. Please fix this issue.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @osbentech , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors
- Good follow-up with the previous reviewer.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — APPROVED** [review]
Hi @osbentech , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well described 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**